src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasPointAndLineSlicer.java

```diff
@@ -164,6 +170,22 @@ private void addRawAtlasPointsToLine(final Location location,
         return result;
     }
 
+    private long createNewPointIdentifier(final PointIdentifierFactory pointIdentifierFactory,
+            final long lineIdentifier)
+    {
+        while (pointIdentifierFactory.hasMore())
+        {
+            final long identifier = pointIdentifierFactory.nextIdentifier();
+            if (this.rawAtlas.point(identifier) == null)
+            {
+                return identifier;
+            }
+        }
+        throw new CoreException(
```

Comment:
Is there a way to print the Atlas' name here too?

Reply:
yup, I'll update for that